docs/SRS/SRS.tex

```diff
@@ -562,11 +562,89 @@ \subsubsection{Speed and Latency Requirements}
 \end{enumerate}
 
 \subsubsection{Safety-Critical Requirements}
-\hspace{1.5cm}N/A
+\begin{enumerate}[start=4,label={PR\arabic*.}]
+	\item Utrition will return an error message when the user uploads an abnormal 
+	image format that is not .png, .jpg, or .jpeg. \\
+	\textbf{Rationale:}  Utrition should not crash by improper user 
+	input. Users should have an opportunity to upload a new file of an appropriate 
+	format.\\	
+	\textbf{Fit Criteria:} Error message should prompt user upon upload of an abnormal 
+	image.
+	
+	\item Utrition will return an error message when the user uploads an image file 
+	that exceeds the maximum size. \\
+	\textbf{Rationale:}  Utrition should not crash by improper user 
+	input. Users should have an opportunity to upload a new file of an appropriate 
+	size.\\	
+	\textbf{Fit Criteria:} Error message should prompt user upon upload of an image 
+	later than 30MB.
+	
+	\item Utrition will return an error message when the user uploads more than three 
+	images at once. \\
+	\textbf{Rationale:}  Utrition should not crash by improper user 
+	input. Users should have an opportunity to upload three or fewer images to the 
+	system.\\	
+	\textbf{Fit Criteria:} Error message should prompt user upon upload of more than 3 
+	images.
+	
+	\item Utrition will prompt the user if food identification cannot 
+	be completed successfully. The user will be notified on the type of error 
+	that occurs. \\
+	\textbf{Rationale:}  Food identification may fail due to a variety of 
+	reasons, and the user should be notified so they may attempt to find a  
+	workaround for the issue. \\	
+	\textbf{Fit Criteria:} Error message should prompt user upon failure to identify 
+	any food items in the uploaded image
+	
+	\item Utrition will return an error message if the request to retrieve 
+	nutritional information cannot be completed successfully. \\
+	\textbf{Rationale:}  Information retrieval requests may fail due to a 
+	variety of reasons, and the user should be notified of the reason why the 
+	service could not be completed as expected.\\	
+	\textbf{Fit Criteria:} Error message should prompt user upon failure to request 
+	nutritional info regarding food items in the uploaded image.
+	
+	\item Utrition will return an error message if the nutritional information 
+	of a specific item cannot be found.\\
+	\textbf{Rationale:}  The user should be notified if the nutritional data of 
+	their food item cannot be fetched.\\
+	\textbf{Fit Criteria:} Error message should prompt user upon failure to find 
+	nutritional info regarding food items in the uploaded image.
+	
+	\item Utrition will return an error message if the user's part nutritional 
+	logs cannot be found.\\
+	\textbf{Rationale:}  The user should be notified if their nutritional data 
+	of past meals cannot be found.\\
+	\textbf{Fit Criteria:} Error message should prompt user upon failure to access 
```

Comment:
Still not sure what this error message is? Are we prompting the user to access their own data? This can't be done if it isn't found

Reply:
Changed error message to notify user that their past data is missing